iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/DirectoryCopySourceCheck.java

```diff
@@ -50,31 +50,31 @@ private static void checkAdd(CheckContext ctx, AddInstruction add) {
     for (Argument src : add.srcs()) {
       String path = ArgumentUtils.resolve(src).value();
       if (path != null && !path.startsWith("http://") && !path.startsWith("https://")) {
-        reportIfSensitive(ctx, src, isSensitivePath(path), "ADD");
+        reportIfSensitive(ctx, src, isSensitivePath(path));
       }
     }
   }
 
   private static void checkCopy(CheckContext ctx, CopyInstruction copyInstruction) {
-    if (hasOption(copyInstruction.options(), "from")) return;
+    if (hasOptionFrom(copyInstruction.options())) return;
```

Comment:
_nit_: what about `hasFromOption`? not sure if it's only me, but `hasOptionFrom` sounds to me too much like `does it have the option from the argument`.